### ISSUE TITLE
"set constraint to current value" doesn't  fully define constraint.

### ISSUE THREAD

**Issue by GrahamRobertsW** [SasView/sasview#2750]
**Describe the bug**
If a parameter is contrained to its current value via right-click "constrain parameter to current value" the constraint isn't fully defined. It doesn't appear in the constraint & simultaneous fit tab. When the project is saved the func attribute and the alue_ex are null in the json. This breaks the constraint if the project is saved and re-opened. Additionally it prohibits further constraints that reference the constrained parameter. 

**To Reproduce**
1. send any curve to fit
2. right click a parameter and select "constrain parameter to current value"
3. open "Constrained or Simultaneous fit" 
4. the constraint is not present

**Expected behavior**
The constraint structure should be completely populated, including the func attribute which should be set. The constraint should be added to the constraint perspective. The constant constraint should be defined as a trivial function that always returns a constant, I believe.

**Screenshots**

**SasView version (please complete the following information):**
 - Version: [e.g. 6.0.0a1]

**Operating system (please complete the following information):**
 - OS: [e.g. MacOS]

**Additional context**
Constraining a value to a constant may have strange behavior with respect to polydispersity or uncertainty.

**Comment by GrahamRobertsW** [comment]
Upon further discussion it is unclear what the intended behavior is. Either the max and min of the fittable parameter should be set to current value +- epsilon, or a constraint object should be created with a trivial function.

**Comment by gonzalezma** [comment]
Yes. The problem is that "Constraint Parameter to its current value" does not create a proper constraint, but simply puts very narrow limits to the values allowed for the selected parameter. The utility of this is not very clear to me and I think that one could remove this option and simply uncheck the parameter box in order to keep it fixed, but others find this option useful.
